support/README.md

```diff
@@ -14,37 +15,59 @@ First response in an hour (during business time); updates to user until either s
 
 Within a day:
 
-- Email inbox (hi@windmill.engineering) (https://groups.google.com/a/windmill.engineering/forum/#!forum/hi)
+- Email inbox (hi@tilt.dev) (https://groups.google.com/a/tilt.dev/forum/#!forum/hi)
 
-- DMs or Mentions to @windmill_eng
+- DMs or Mentions to (@tilt_dev)[https://twitter.com/tilt_dev]
 
 - All new users should be researched and entered into the CRM. If they’re a
   promising lead, the exterminator should reach out to them.
 
-For any new bugs reported by users, the Exterminator should create a Github
-issue. For any existing bugs reported by users, the Exterminator should note the
-request in the existing Github issue / direct the user to the existing issue and
-ask them to +1 it so we know who to follow up with,
+### Support Tracking
 
-Anyone with a bug to report should slack it to #triage-bugs and, if it turns out
-to be a new bug, file an issue in GitHub . Issues with a bug label are
+When users request support, they may be reporting a bug, requesting a feature, asking
+for docs/examples, or vaguely describing a general problem that they're having
+trouble solving with Tilt.
+
+These user support requests may lead to work that we need to track.
+
+We have two systems for tracking this work:
+
+- GitHub issues for all public, user-reported issues
+
+- Clubhouse for internal tracking and prioritization
+
+GitHub issues that we are actively working on should be tracked in
+Clubhouse. The reverse is not true - we may internal work
+that is not expressed in GitHub issues.
+
+### Support Triage
+
+When a user reports an issue, the Exterminator should ensure there's a Github issue.
+
+The exterminator should confirm with the user that the issue write-up accurately
+expresses their problem, and ask them to +1 it or comment with additional context.
+
+We use these votes and comments to decide which issues to prioritize. That
+_doesn't_ mean "the issue with the most votes wins". It's just one variable we
+use in prioritization.
+
+Issues with a bug label are
```

Comment:
as per [this comment](https://github.com/windmilleng/company/pull/3#issuecomment-592150284)/our new definition of "bug", a. this doc should probably be more deliberate about "support request" instead of bug", and b. is it time to rethink the "bug" vs "enhancement" label?

Reply:
meaning: if it's still baked into our process that the only things we care about fixing asap are "bugs" then we'll either be mislabeling bad UX as bugs in order to fix it, or not fixing bad UX cuz it's not technically a "bug".

(I guess you could make the case that the Exterminator SHOULD only worry about triaging "bugs" [as defined here](https://github.com/windmilleng/company/tree/master/development#bug), but I can think of some things that are poor UX and merit immediate fixing that aren't technically bugs by this definition--e.g. a new feature is causing problems for a specific setup and we need to add a way for users to turn it off, or our logs for something are misleading)